src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalAnalysisResult.js

```diff
@@ -413,7 +446,13 @@ const ClinicalAnalysisResult = ({
               />
             </Row>
             <Column style={{ marginTop: 10 }}>
-              <ControlPanelNode name={'ExploreCases'} analysis_id={id} />
+              <ControlPanelNode
+                parsedFacets={parsedFacets}
```

Comment:
parsedFacets is not used in ControlPanelNode. So remove it?

Reply:
👍